assets/check

```diff
@@ -15,6 +15,8 @@ suffix=$(echo "${payload}" | jq -r '.source.suffix // ""')
 endpoint_url="$(echo "$payload" | jq -r '.source.endpoint // ""')"
 region="$(echo "$payload" | jq -r '.source.region // "us-east-1"')"
 use_v4="$(echo "$payload" | jq -r '.source.use_v4 // ""')"
+skip_ssl_verification="$(echo "$payload" | jq -r '.source.skip_ssl_verification // ""')"
+ca_bundle="$(echo "$payload" | jq -r '.source.skip_ssl_verification // ""')"
```

Comment:
We're setting ca_bundle here, but we're fetching the skip_ssl_validation field from the payload. Is that correct? It looks like a cut 'n' paste that didn't get edited.

Reply:
Fixed.